runtime/oti/j9nonbuilder.h

```diff
@@ -5025,6 +5026,7 @@ typedef struct J9InternalVMFunctions {
 	BOOLEAN (*jvmRestoreHooks)(struct J9VMThread *currentThread);
 	BOOLEAN (*isCRIUSupportEnabled)(struct J9VMThread *currentThread);
 	BOOLEAN (*isCRIUSupportEnabled_VM)(struct J9JavaVM *vm);
+	BOOLEAN (*disableCRIUSecProvider)(struct J9VMThread *currentThread);
```

Comment:
This name is now wrong; it should be `enableCRIUSecProvider` to match the function to which it refers.

Reply:
Updated